.github/workflows/linter.yml

```diff
@@ -5,6 +5,9 @@ name: Linter
 on:
   pull_request:
     branches: [ main ]
+    paths:
+      - '**.json'
+      - '**.md'
```

Comment:
This prevents running linter when only shell file have changes

Reply:
Does that matter if only the JSON and Markdown files are linted?

If so, is there a certain file extension that needs to be included?

Reply:
https://github.com/actions/virtual-environments/blob/499a9c967ffc320d970b964f57230065d08aa1f5/.github/workflows/linter.yml#L32-L34

Reply:
It now also runs on changes to `.sh` files, which the shebang linter checks. Thanks!